assets/js/main.js

```diff
@@ -106,14 +99,6 @@ function selectImagePart(element) {
 
 
 const multichoiceAnswers = document.querySelectorAll('.multichoice-answer');
-// for (let i = 0; i < multichoiceBtns.length; i++) {
```

Comment:
I really like the way you refactored this piece of code.

Reply:
Thanks! 🙌🎉🦕